packages/core/stories/tokens/logo/_logo-display.scss

```diff
@@ -0,0 +1,8 @@
+@use "00-base/configure" as *;
+@use "sass:color" as *;
```

Comment:
why is this needed?

Reply:
storybook doesn't encapsulate styles well, this will override the .sg-swatch class in colors